src/components/Home.tsx

```diff
@@ -21,8 +20,8 @@ import {
   color,
 } from "styled-system";
 import BuyButton from "./BuyButton";
+import MenuButton from "./NavMenu";
```

Comment:
Maybe import this as "NavMenu"? Feels clearer to me, whaddayareckon?

Reply:
Got myself all mixed up with buttons and navs and whatnot. This is all fixed in the other tickets but have updated here anyway.